script/universal/MultisigBuilder.sol

```diff
@@ -48,12 +49,12 @@ abstract contract MultisigBuilder is MultisigBase {
      * Alternatively, this method can be called by a threshold of signers, and those signatures
      * used by a separate tx executor address in step 2, which doesn't have to be a signer.
      */
-    function sign() public view returns (bool) {
```

Comment:
Is it OK to remove returns from all the public functions?

Reply:
Touched on this in the PR description a bit but it seemed like they were never used, and all methods should now revert if a transaction fails, so my thinking was they don't add much value